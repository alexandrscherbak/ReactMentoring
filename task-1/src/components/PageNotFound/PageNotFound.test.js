import React from 'react';
import Enzyme, { mount } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import PageNotFound from './PageNotFound.jsx';
import renderer from 'react-test-renderer';

Enzyme.configure({ adapter: new Adapter() });

function setup() {
	return mount(<PageNotFound/>);
}

describe('PageNotFound', () => {
	test('snapshot', () => {
		const component = renderer.create(
			<PageNotFound/>,
		);
		let tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});
	it('should render self and subcomponents', () => {
		const enzymeWrapper = setup();
		expect(enzymeWrapper.find('h1').text()).toBe('Page Not Found');
	});
});
