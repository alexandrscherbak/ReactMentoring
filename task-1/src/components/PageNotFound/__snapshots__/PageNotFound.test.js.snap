// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound snapshot 1`] = `
<h1>
  Page Not Found
</h1>
`;
